packages/nextjs/styles/globals.css

```diff
@@ -27,9 +27,28 @@ p {
   @apply shadow-md;
 }
 
+html {
+  @apply overflow-y-scroll;
+}
+
+body {
+  @apply overflow-y-auto;
+}
```

Comment:
can consider hiding overflow x? what do you think?

Reply:
Hi @Kingsuite please remember the required changes

Reply:
GM @Nadai2010 @metalboyrick, sorry for the delay. Changes have been implemented. Thanks for your review